resources/views/projects.blade.php

```diff
@@ -49,6 +49,19 @@
 
 @section('scripts')
 <script>
+	function adjustProjectsGrid(maxWidth) {
```

Comment:
Big nono. El CSS framework hace esto por ti:

https://semantic-ui.com/collections/grid.html#doubling o en todo caso los "wide device" que vienen ah&iacute; abajo

Reply:
oh cool - totally missed that. thanks!